client/lib/keyboard-shortcuts/key-bindings.js

```diff
@@ -1,201 +1,174 @@
 /** @format */
 
-/**
- * External dependencies
- */
+// Returns a function that returns a translated text when applied
+const translate = text => tr => tr( text );
```

Comment:
Can we rename it somehow? People might confuse it with the `translate` from `i18n-calypso`.

Reply:
You mean the `translate` function? I think it's actually very important that it has exactly this name -- there is a l10n script that searches the sources for instances of `translate( 'something' )` and generates the list of strings to translate. @akirk or @yoavf can confirm if that's correct or not.

Reply:
Correct, we explicitly look for the function call named `translate`, see https://github.com/Automattic/i18n-calypso/blob/master/cli/index.js#L20 resp. https://github.com/Automattic/xgettext-js/blob/master/xgettext.js#L214